src/Nordea/Components/FlatLink.elm

```diff
@@ -103,6 +109,47 @@ view attributes children (FlatLink config) =
         children
 
 
+inline : String -> String -> Html msg
+inline url text =
```

Comment:
I suggest an interface like this. Makes it possible to add aria-tags, onclick tracking etc. on demand

```suggestion
inline : List (Attribute msg) -> List (Html msg) -> Html msg
inline attrs children =
```